docs/tools/vdb_table/data/myscale.json

```diff
@@ -168,7 +168,7 @@
   },
   "npm_downloads": {
     "value": 0,
-    "source_url": "",
+    "source_url": "https://www.npmjs.com/package/@clickhouse/client",
```

Comment:
My scale file, not clickhouse.

Reply:
Interesting - well, I think the purpose of the stats is to track the popularity of a specific solution. And even though myscale uses the actual clickhouse client, attributing the clickhouse client popularity to myscale doesn't seem right.